From 77057119abf5d6f17ab34c1903db8e8d0d894047 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Mon, 9 Dec 2013 11:35:03 +0000
Subject: [PATCH 0501/1077] change Mantis 0030063: mcontrol: do not allow to
 close mcontrol without starting explorer when started as user shell

mControl.exe
V1.0.2.13



git-svn-id: svn://localhost/SelfServiceCommon/trunk@621 90b65887-3827-0410-9a23-83215b262276
---
 .../Massai/cpp/Controller/inc/mcontrol_args.h    | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)
 create mode 100644 SelfServiceCommon/Massai/cpp/Controller/inc/mcontrol_args.h

diff --git a/SelfServiceCommon/Massai/cpp/Controller/inc/mcontrol_args.h b/SelfServiceCommon/Massai/cpp/Controller/inc/mcontrol_args.h
new file mode 100644
index 00000000..f22b7c8e
--- /dev/null
+++ b/SelfServiceCommon/Massai/cpp/Controller/inc/mcontrol_args.h
@@ -0,0 +1,16 @@
+// arguments for mcontrol
+
+#ifndef MCONTROL_ARGS_H_INCLUDED
+#define MCONTROL_ARGS_H_INCLUDED
+#ifdef SHOW_INCLUDE_FILES
+#pragma message("including: " __FILE__)
+#endif
+
+  HINSTANCE getInstance();
+  bool serverMode();
+  bool noLOG();
+  bool noSNMP();
+  bool portableMCC();
+  bool isRunningAsUserShell();
+
+#endif
-- 
2.41.0.windows.1

